.api-testing.config.json

```diff
@@ -0,0 +1,4 @@
+{
+    "base_uri": "",
+    "main_page": "Main_Page"
```

Comment:
You don't need the main_page info

Reply:
The "Main_Page" info has been removed from `.api-testing.config.json` file as per your advice.